java-generator/it/src/it/extensions/src/test/java/io/fabric8/it/extensions/camelk/JavaGeneratedCamelK.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.fabric8.it.extensions.camelk;
+
+import org.apache.camel.v1.CamelCatalog;
+import org.apache.camel.v1.CamelCatalogSpecBuilder;
+import org.apache.camel.v1.camelcatalogspec.ArtifactsBuilder;
+import org.apache.camel.v1.camelcatalogspec.RuntimeBuilder;
+import org.apache.camel.v1.camelcatalogspec.artifacts.dependencies.ExclusionsBuilder;
+import io.fabric8.kubernetes.api.model.ObjectMetaBuilder;
+
+public class JavaGeneratedCamelK {
+
+  // Code generated with
+  // ./java-generator/cli/target/java-gen -dest="java-generator/it/src/main/java" -crd=java-generator/it/src/it/extensions/src/resources/camel-k.crds.1.8.1.yaml --add-extra-annotations
+
+  public static CamelCatalog dummyCamelKCatalog() {
+    CamelCatalog camelCatalog = new CamelCatalog();
+    camelCatalog.setMetadata(
+      new ObjectMetaBuilder()
+        .withName("dummy")
+        .build()
+    );
+    camelCatalog.setSpec(
```

Comment:
This was a good exercise overall, and here we are catching the fact that we miss `sundrio` annotations on the `CustomResource` itself, cc. @manusa (I'll follow up with a separate PR)

Reply:
fixed in this PR.